mssqlcli/jsonrpc/contracts/queryexecutestringservice.py

```diff
@@ -23,50 +26,51 @@ def get_response(self):
             Get latest response, event or exception if occured.
         """
         try:
-            response = self.json_rpc_client.get_response(self.id, self.owner_uri)
+            response = self.json_rpc_client.get_response(self.request_id, self.owner_uri)
 
             decoded_response = None
             if response:
                 decoded_response = self.decode_response(response)
```

Comment:
Given this is now static, should't we use the class and not `self` to call it?

Reply:
Interesting question... I suppose we could. Would there be any effective difference? Genuinely curious here.

Reply:
After some thought, I like your idea of referencing the class instead of `self` for static methods. For readability's sake, it makes it clear and obvious that it's a static method being used.